### ISSUE TITLE
Log In drop down text should use consistent wording with button text and improve button accessibility

### ISSUE THREAD

**Issue by deCarina** [unstructuredstudio/zubhub#979]
Issue
1. when attempting to log in a modal shows
There is an inconsistency between the "Sign Up or Log In to your ZubHub account" text and the 2 buttons Sign Up and Sign In

2. Accessibility of buttons should be improved

To Reproduce
Go to https://zubhub.unstructured.studio  
log out
then try to log in

Expected behaviour
1. Sign In button should be consistent with the drop-down text and use Log In
2. Buttons should use darker backgrounds defined by the style guide to improve accessibility

![screenshot_US](https://github.com/unstructuredstudio/zubhub/assets/149316020/3cd7eb0f-a92c-4629-a02e-38ed7eade21c)
Perhaps this is a minor UX issue, but as a user, I was bothered that the wording in the drop-down text was not consistent with the button text. Sign In on a button should be replaced by Log In. Sign Up and Sign In could be confusing words to non-native English audiences

<img width="1056" alt="Screenshot 2023-10-29 at 22 23 26" src="https://github.com/unstructuredstudio/zubhub/assets/149316020/34bd1ae7-41e2-4999-b70f-ee54e0f1dc8e">

Here the instruction text to Log In is consistent with the button text to Log In.
.
<img width="1068" alt="Screenshot 2023-10-29 at 22 27 40" src="https://github.com/unstructuredstudio/zubhub/assets/149316020/73bb2920-1a74-4888-9756-3d469ea98949">

I think the buttons in this instance do not meet the accessibility guidelines as far as color contrast and should use the darker color that is currently defined in the style guide. I realize this may cause a major work throughout the web app.

@srish and @tuxology  I would appreciate your comments.

@DonPresh you talk about accessibility improvements in #816. What is your opinion on these buttons?

Screenshots

**Comment by srish** [comment]
@deCarina Thanks a lot for your suggestions! While I agree with your idea, from what I know, small enhancements like these will be prioritized in much later phases of the project as part of a larger goal https://github.com/unstructuredstudio/zubhub/issues/816. I'm closing this for now.

**Comment by deCarina** [comment]
@srish thank for your feedback and comments of explanation.
